### PR TITLE
Update announcements documentation

### PR DESCRIPTION
**Pull request by jflory7**
# Summary

* This is a…
    * [ ] Bug fix
    * [ ] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [X] Other
* **Describe this change in 1-2 sentences**: Follow RST style guide, make minor English edits, add examples for announcements

# Problem

Documentation is somewhat outdated, needs to be updated

# Solution

* Uses ReStructuredText style guide for the announcements documentation
* Uses active voice / minor edits
* Adds examples

This was tested locally in a development environment.

# Action

Review for content, and if it looks good, merge. :) Once merged, it will go live [here]().

### CODE REVIEW

File: docs/user/announcements.rst [hunk 1]
```diff
@@ -2,22 +2,63 @@
...
+quotes (``''``). MobArena ignores announcements set to an empty value. To
+disable one, you must override it. Deleting the option does not disable an
+announcement since MobArena adds them in by default.
+
+
+********
+Examples
+********
+
+Three examples below show all of the features explained above.
```
Comment (garbagemule) [inline docs/user/announcements.rst]: The examples don't seem to be shown here on github in the preview. Is that a bug in the markdown or a "missing feature" of github? I've seen code blocks in other places o.O

Comment (jflory7) [reply docs/user/announcements.rst]: @garbagemule The code block doesn't render on GitHub, but it will on ReadTheDocs. This type of code block is something that is either unique to Sphinx or ReadTheDocs, I can't remember which. If you run `make clean html` or the `Make.bat` file on Windows, it should build and render the docs in HTML.

tl;dr: It will look correct in the RTD site. :)

Comment (garbagemule) [reply docs/user/announcements.rst]: Roger :)
